tests/phpunit/test-wc-stripe-sub-renewal.php

```diff
@@ -39,6 +46,23 @@ public function setUp() {
 			->will(
 				$this->returnValue( true )
 			);
+
+		$this->statement_descriptor = 'This is a statement descriptor.';
+		add_option(
+			'woocommerce_stripe_settings',
+			array(
+				'statement_descriptor' => $this->statement_descriptor,
+			)
+		);
+	}
+
+	/**
+	 * Tears down the stuff we set up.
+	 */
+	public function tearDown() {
+		delete_option( 'woocommerce_stripe_settings' );
+
+		parent::tearDown();
```

Comment:
Small nitpick: Unless there is a need to prepare data for the parent constructor, let's call it in the beginning of the method.

Reply:
There isn't a need per se, it's just that if the tests fail for any reason, we don't have to write `catch` code in order to make sure that we `delete_option` at the end of the test run.

Reply:
Isn't `tearDown` being called irregardless of whether the tests fail or succeed?

Reply:
Ah, I gotcha now 👌 Done.